src/App.tsx

```diff
@@ -1,18 +1,144 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import {
+  useContext, useEffect, useMemo, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import {
+  getTodos, addTodo, deleteTodo, updateTodoStatus, updateTodoTitle,
```

Comment:
this is not a mistake, you can also write this way, just a recommendation

in large projects it is convenient when each variable is on a new line, it is convenient to delete or copy through shortcuts, and also in the git there are not so many changes (when editing a method only one line)
```suggestion
  getTodos,
  addTodo, 
  deleteTodo, 
  updateTodoStatus, 
  updateTodoTitle,
```